### ISSUE TITLE
Config option for spacing

### ISSUE THREAD

**Issue by jonathon-boldcommerce** [nvuillam/markdown-table-formatter#10]
The formatter currently expects this:
```
| First Header | Second Header |
|--------------|---------------|
| Content Cell | Content Cell  |
| Content Cell | Content Cell  |
```
but should have the option to allow this:
```
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |
```

**Comment by nvuillam** [comment]
This would be something to ask to https://github.com/darkriszty/MarkdownTablePrettify-VSCodeExt#readme , the tool behind markdown-table-formatter ^^

**Comment by jonathon-boldcommerce** [comment]
https://github.com/darkriszty/MarkdownTablePrettify-VSCodeExt/issues/62

**Comment by nvuillam** [comment]
@jonathon-boldcommerce available in v1.3.0 :)

https://github.com/nvuillam/markdown-table-formatter/releases/tag/v1.3.0
